diff -urN aa/libgconf-java-2.8.0/src/Makefile.in libgconf-java-2.8.0/src/Makefile.in
--- aa/libgconf-java-2.8.0/src/Makefile.in	2004-09-16 14:31:23.000000000 +0200
+++ libgconf-java-2.8.0/src/Makefile.in	2004-11-02 01:40:39.677079128 +0100
@@ -86,7 +86,7 @@
 		$<
 	
 build-java/%.o: java/%.java
-	$(GCJ) $(GCJFLAGS) -g -O --classpath=$(JAVA_SOURCE_PATH):${GCJ_CLASSPATH} -o $@ -c \
+	$(GCJ) $(GCJFLAGS) -g -O --classpath=$(JAVA_SOURCE_PATH):${GCJ_CLASSPATH}:${CLASSPATH} -o $@ -c \
 		$<
 		
 COMPILED_CLASSES:=$(shell echo $(CLASSES) | sed -e 's=\.class=\.o=g' )
