diff -urN aa/libgconf-java-2.5.4/src/Makefile.in libgconf-java-2.5.4/src/Makefile.in
--- aa/libgconf-java-2.5.4/src/Makefile.in	2004-02-04 01:55:18.000000000 +0100
+++ libgconf-java-2.5.4/src/Makefile.in	2004-02-08 22:51:53.112052824 +0100
@@ -62,6 +62,8 @@
 GNOME_LIBS=@GNOME_LIBS@
 LIBGLADE_CFLAGS=@LIBGLADE_CFLAGS@
 LIBGLADE_LIBS=@LIBGLADE_LIBS@
+GCONF_CFLAGS=@GCONF_CFLAGS@
+GCONF_LIBS=@GCONF_LIBS@
 GCJ=@GCJ@ -fPIC -fjni
 GCJFLAGS=
 GCJ_CLASSPATH=@GCJ_CLASSPATH@.
@@ -69,7 +71,7 @@
 .SUFFIXES: .java .class
 
 .c.o:
-	$(CC) -c $(CFLAGS) $< $(JAVA_INCLUDES) $(GTK_CFLAGS) \
+	$(CC) -c $(CFLAGS) $< $(JAVA_INCLUDES) $(GTK_CFLAGS) $(GCONF_CFLAGS) \
 	    $(LIBGTKHTML_CFLAGS) $(GNOME_CFLAGS) $(LIBGLADE_CFLAGS) \
 		\
 		-o $@
